### PR TITLE
Fixed wording a bit

### PR DESCRIPTION
**Pull request by SimonVerhoeven**
- [x] The page (if new), does not already exist in the repo.
- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#guidelines).
- [x] The page description includes a link to documentation or a homepage (if applicable).

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -20,8 +20,8 @@
 [license-url]: https://github.com/tldr-pages/tldr/blob/master/LICENSE.md
 [license-image]: https://img.shields.io/github/license/tldr-pages/tldr.svg
 
-Contributions to the tldr-pages project are [most welcome](GOVERNANCE.md)!
-All `tldr` pages are stored in Markdown right here on GitHub.
+Contributions to the TLDR-pages project are [most welcome](GOVERNANCE.md)!
```
Comment (schneiderl) [inline CONTRIBUTING.md]: I'm unsure about this. 
Should we have this in uppercase? @sbrl @mebeim @waldyrious

Comment (mebeim) [reply CONTRIBUTING.md]: Not really sure.. but this suggested change is inconsistent: it only changes some occurrences of the word, not all of them. :thinking:. I don't personally have a preference between the two by the way.

Comment (schneiderl) [reply CONTRIBUTING.md]: I think I have an opinion now.
"TLDR" is the term while "tldr" is the project name. When we're talking about the project, I believe we should keep it lower case.

Comment (mebeim) [reply CONTRIBUTING.md]: That could be a good point.

Comment (sbrl) [reply CONTRIBUTING.md]: I believe that a while ago it was decided that `tldr-pages` is the official wording for referring to the tldr-pages project.

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -30,14 +30,14 @@ To get started, please [sign](https://cla-assistant.io/tldr-pages/tldr) the
 
 ## Guidelines
 
-The basic format of a `tldr` page is a set of concrete usage examples.
+The basic format of a `TLDR` page is a set of concrete usage examples.
```
Comment (sbrl) [inline CONTRIBUTING.md]: Again here. `tldr` should stay as `tldr`, not `TLDR`.

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -91,7 +91,7 @@ Some examples of valid locale tags:
 
 ### Default language for newly added pages
 
-The default language used for pages is English (US). Pages written in English are stored in the default `pages` directory (notice the absence of a specific language tag). Although not strictly required, if you'd like to add a new page in a different language, please consider creating the English page too.
```
Comment (sbrl) [inline CONTRIBUTING.md]: I'm not sure this change makes it clearer. The `used` is important, as it indicates that the language is used on a pages.

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -116,7 +116,7 @@ Examples:
 
 ## Licensing
 
-`tldr` is licensed under the [MIT license](https://github.com/tldr-pages/tldr/blob/master/LICENSE.md).
+`TLDR` is licensed under the [MIT license](https://github.com/tldr-pages/tldr/blob/master/LICENSE.md).
```
Comment (sbrl) [inline CONTRIBUTING.md]: And again here. This should stay as-is.

File: CONTRIBUTING.md [hunk 5]
```diff
@@ -30,14 +30,14 @@ To get started, please [sign](https://cla-assistant.io/tldr-pages/tldr) the
...
-The basic format of a `tldr` page is a set of concrete usage examples.
+The basic format of a `TLDR` page is a set of concrete usage examples.
 Here are a few guidelines to get started:
 
 1. Try to keep pages at around 5 examples. Pages can be longer if needed, but don't exceed 8 examples.
    Remember, it's OK if the page doesn't cover everything; that's what `man` is for.
 2. When in doubt, keep new command-line users in mind. Err on the side of clarity rather than terseness.
    For example, commands that require `sudo` should include it directly in the examples.
-3. Try to incorporate the spelled-out version of single-letter options in the example's description.
+3. Try to incorporate the spelt-out version of single-letter options in the example's description.
```
Comment (sbrl) [inline CONTRIBUTING.md]: Nice catch.

Comment (agnivade) [reply CONTRIBUTING.md]: Except this change, let's revert all the others.

File: CONTRIBUTING.md (hunk 1 above)
Comment (waldyrious) [reply CONTRIBUTING.md]: > I believe that a while ago it was decided that tldr-pages is the official wording for referring to the tldr-pages project.

IIRC we didn't end up making a final decision. We just recently restarted discussion in #1109, but let me know if there are other discussions I'm missing.

In any case, I'm pretty sure all-caps TLDR was never widely supported in such discussions, so this particular bit should be reverted.

File: CONTRIBUTING.md (hunk 5 above)
Comment (waldyrious) [reply CONTRIBUTING.md]: It's not actually incorrect, it's just [one of the valid variants](https://en.wiktionary.org/wiki/spelled) for the past participle of "spell".

**Comment by waldyrious** [comment]
Considering the comments left inline, it looks like none of the proposed changes are mergeable. Thanks for taking the time to contribute, @SimonVerhoeven, but regrettably we'll have to close the PR unmerged.

**Pull request closed without merging**